src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -226,6 +228,26 @@ public Builder addMember(String name, CodeBlock codeBlock) {
       return this;
     }
 
+    public Builder addMember(String name, Object value) {
```

Comment:
Or make it package private - the static factory `AnnotationSpec get(Annotation annotation, boolean includeDefaultValues)` needs it, but it does not have to be public.